custom/aaa/static/aaa/spec/filters/beneficiary_type.spec.js

```diff
@@ -0,0 +1,33 @@
+/* global sinon */
```

Comment:
Why is `global sinon` required here? I don't see any usage in a some of these files